Makefile

```diff
@@ -1,6 +1,8 @@
 include common.mk
 MODULES=dss tests
 
+all: test
```

Comment:
Can you just bring the test target up top instead of declaring a new one?

Reply:
Will do.